### PR TITLE
Fixing fingerprints within plugins

### PR DESCRIPTION
**Pull request by 0xInfection**
> __NOTE:__ This is a __work in progress__ pull request. I'll ping when this is complete.

This pull request fixes and updates fingerprints in the plugins already existing within wafw00f. More commits to come up as I go on fixing more.

__UPDATE:__ The work on this branch is over, feel free to review and merge.

### CODE REVIEW

**Comment by sandrogauci** [comment]
just noticed your WIP comment after doing some basic review. Will review again once you're done

**Comment by 0xInfection** [comment]
I made this PR so that you can keep reviewing as I keep on pushing the changes. :)

File: wafw00f/plugins/radware.py [hunk 1]
```diff
@@ -15,4 +15,7 @@ def is_waf(self):
             b'<TITLE>Unauthorized Request Blocked</TITLE>', b'If you believe that there has been some mistake',
             b'?Subject=Security Page - Case Number')):
             return True
+        # Restored a fingerprint for radware previously discarded
+        if b'CloudWebSec@radware.com' in responsebody:
```
Comment (0xInfection) [inline wafw00f/plugins/radware.py]: This new fingerprint is more reliable. It was earlier discarded due to false negatives.

**Comment by sandrogauci** [comment]
cool.. so we're down to 47 now :laughing: but much better checks overall and definitely updated. Most checks require more requests now in general (it seems) but they're also more reliable. There still is plenty of room for improvement I suppose but good to merge.

**Comment by 0xInfection** [comment]
Don't worry you've another 50+ WAFs coming up next. 😉

**Pull request merged**